README.md

```diff
@@ -1,4 +1,61 @@
 
 ## Overview
 
-The `enhancements` repo is made available to discuss specific new features or architectural improvements for the community around `open-cluster-management`.
\ No newline at end of file
+The `enhancements` repo is made available to discuss specific new features or architectural improvements for the community around `open-cluster-management`.
+
+Inspired by the [Kubernetes enhancement]{https://github.com/kubernetes/enhancements} process.
+
+This repository provides a rally point to discuss, debate, and reach consensus for how Open Cluster Management enhancements are introduced.
+
+Enhancements may take multiple releases to ultimately complete and thus provide the basis of a community roadmap. Enhancements may be filed from anyone in the community, but require consensus from domain specific project maintainers in order to implement and accept into the release.
+
+For template references, see the [template]{https://github.com/open-cluster-management/enhancements/blob/master/guidelines/enhancement_template.md}.
+
+## Is My Thing an Enhancement?
+
+A rough heuristic for an enhancement is anything that:
```

Comment:
I think the blog post criteria from kube is a pretty good one.  If it's announcement worthy, it's probably enhancement worthy.